data/tiles.yml

```
@@ -2,12 +2,13 @@
 test_analytics_features:
   - title: "Deep performance analysis"
     url: "/docs/test-analytics/test-suites#trends-and-analysis"
-    image_path: "images/test_analytics/icon-performance-analysis.svg"
     desc: "Automatic tracing across your test suite, deeply integrated with your programming language and test framework."
   - title: "Find and fix flaky tests"
     url: "/docs/test-analytics/test-suites#detecting-flaky-tests"
-    image_path: "images/test_analytics/icon-find-flaky-tests.svg"
     desc: "Quickly identify which tests are the most disruptive for your team, and get a head-start on fixing them."
+  - title: "See urgent issues in your run"
+    url: "/docs/test-analytics/test-suites#run-issues"
+    desc: "Find problematic tests in a given run and resolve their issues."
```

If this is factually correct then these words sounds better:

```suggestion
    desc: "Find and resolve issues with problematic tests in every run."
```